frontend/test/metabase/scenarios/admin/datamodel/metrics.cy.spec.js

```diff
@@ -263,5 +263,22 @@ describe("scenarios > admin > datamodel > metrics", () => {
       cy.findByText("13022_Metric"); // Name
       cy.findByText("Orders, CE"); // Definition
     });
+
+    it("should show CE that uses 'AND/OR' (metabase#13070)", () => {
```

Comment:
This is adopted from @nemanjaglumac's PR in #13560.

Reply:
Should I close #13560?

Reply:
Yeah, I think it's safe to close #13560.